alphagov/paas-cf
# Update schmie signature GPG key fingerprint

What
----

~Removing main key fingerprint and adding in subkey for Sebastian Schmieschek.
Providing the main key as public key for a signature subkey when verifying commit signatures is no longer working.~
At some point along the way something got mangled for pubkey 1CD403234CB4354F0E1C1C8F409948B4408E4860 and currently merge sign checks fail for @schmie. Re-running `make update_merge_keys` seems to do the trick to fix.

How to review
-------------

- Run `make dev pipeline SKIP_COMMIT_VERIFICATION=false` in dev env and check the pipeline-log "get paas-cf" step is happy with a commit signed by @schmie (e.g. this branch).
- Make sure other merge-keys (e.g. yours) still work.

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨
